### PR TITLE
Add css-modules example

### PR DESCRIPTION
**Pull request by just-boris**
This is not a final version, but at least shows the general conception – all class names are dynamically generated. To check this out, you need to run `npm install && npm run build` in `css-modules` folder.

Work in progress. To be done

* [x] add build step to render static markup. (Currently, it is being rendered in the browser)

/cc @sapegin for review as well.

### CODE REVIEW

File: css-modules/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import App from "./app";
+
+function render(container) {
+  container.appendChild(App());
+}
+
+document.addEventListener("DOMContentLoaded", () => {
```
Comment (sapegin) [inline css-modules/index.js]: You don’t really need this event here ;-)

**Review by sapegin — COMMENTED** [review]
This is actually very good! I’d make something much more complicated. Probably closer to real life, but probably worse as an example ;-)

**Comment by AleshaOleg** [comment]
@just-boris thanks for your pull request! @sapegin also thanks to you for review! Would you like to make some changes to this example? Or I can merge it?

**Comment by just-boris** [comment]
I am going to add some more fixes, so it will be working without Javascript as well as all other examples.

**Comment by AleshaOleg** [comment]
@just-boris nice! So, after this I will add to other examples link to this demo, and will merge it:)

**Comment by just-boris** [comment]
Finished.

Was happy to contribute!

**Comment by AleshaOleg** [comment]
@just-boris thanks to you! Now, I will add links in other examples and will deploy to demo page.

**Comment by AleshaOleg** [comment]
@just-boris could you tell me, how I can run it? Already, installed npm packages, and ran `webpack`. It's successfully builded, but I need to have an index.html to install this demo. Or you think it will be enough, just a code example to see how to works with css-modules?

**Comment by just-boris** [comment]
You need to run `npm run build`. It runs not only the webpack, but also runs the script that webpack had built for you.

Look at the `package.json`, it contains all definitions for that

**Comment by AleshaOleg** [comment]
@just-boris thanks, already deployed you demo:) [Here](http://aleshaoleg.me/holy-grail-markup/css-modules/) it is

**Pull request merged**